code/modules/mob/living/simple_animal/abnormality/he/singing_machine.dm

```diff
@@ -72,54 +97,60 @@ Finally, an abnormality that DOESN'T have to do any fancy movement shit. It's a
 			bonusRed = minceRed // Should be stronger, for when working it is EXTRA dangerous.
 			manual_emote("makes a horrible grinding noise!") // Oh boy, it's mad.
 			playsound(src, 'sound/abnormalities/singingmachine/chew.ogg', 60, 0, 3)
-	else if(get_attribute_level(user, FORTITUDE_ATTRIBUTE) >= 80 || get_attribute_level(user, TEMPERANCE_ATTRIBUTE) < 60)
-		statChecked = 1 // I see you've failed one of the stat checks, but have also chosen not to feed yourself to the machine.
+		return ..()
+
+	// I see you've failed the stat check, but have also chosen not to feed yourself to the machine.
+	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) <= 40)
```

Comment:
This check is less accurate than the previous one for "2 and lower" as you're still level 2 in a stat until you hit 60.
```suggestion
	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) < 60)
```